desktop/libs/librdbms/src/librdbms/jdbc.py

```diff
@@ -96,6 +97,12 @@ def connect(self):
     if self.conn is None:
       self.gateway.jvm.Class.forName(self.jdbc_driver)
       self.conn = self.gateway.jvm.java.sql.DriverManager.getConnection(self.db_url, self.username, self.password)
+      if 'presto' in self.jdbc_driver:
```

Comment:
Shouldn't this be done in desktop/libs/notebook/src/notebook/connectors/jdbc_presto.py?

Reply:
No, It's the different purpose of jdbc_presto.py - it defined helper methods. There is no class that inherit Jdbc class